WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -442,13 +442,16 @@ private async Task CreateRoundsAsync(CancellationToken cancellationToken)
 
 				var roundWithoutThis = Rounds.Except(new[] { round });
 				RoundParameters parameters = RoundParameterFactory.CreateRoundParameter(feeRate, largeSuggestion);
+
+				// Use an existing large round instead of creating a new one if there is one with enough time remaining
+				// and not already too crowded to support inputs inflow from next to be destroyed round
 				Round? foundLargeRound = roundWithoutThis
 					.FirstOrDefault(x =>
 									x.Phase == Phase.InputRegistration
 									&& x is not BlameRound
-									&& !x.IsInputRegistrationEnded(Config.MaxInputCountByRound)
+									&& x.InputCount < 50 || roundDestroyerInputCount * 0.7 <= roundDestroyerInputCount - x.InputCount / (1 - (x.InputRegistrationTimeFrame.Remaining.TotalSeconds / x.InputRegistrationTimeFrame.Duration.TotalSeconds))
```

Comment:
I guess inserting parenthesis as following would solve the CF issue:

```suggestion
									&& (x.InputCount < 50 || roundDestroyerInputCount * 0.7 <= roundDestroyerInputCount - x.InputCount / (1 - (x.InputRegistrationTimeFrame.Remaining.TotalSeconds / x.InputRegistrationTimeFrame.Duration.TotalSeconds)))
```